spec.html

```diff
@@ -38107,7 +37942,7 @@ <h1>
         <emu-clause id="sec-allocatetypedarraybuffer" type="abstract operation">
           <h1>
             AllocateTypedArrayBuffer (
-              _O_: a TypedArray object,
+              _O_: a TypedArray that has a [[ViewedArrayBuffer]] internal slot,
```

Comment:
```suggestion
              _O_: a TypedArray,
```

All TypedArrays have [[ViewedArrayBuffer]].